roles/geth/templates/genesis.json

```diff
@@ -1,20 +1,796 @@
 {
-    "config": {
-        "chainId": {{ networkid }},
-        "homesteadBlock": 0,
-        "eip155Block": 0,
-        "eip158Block": 0,
-        "byzantiumBlock": 0
-    },
-    "alloc": {
-        "{{ alloc_address }}": {"balance": "10000000000000000000000"}
-    },
-    "coinbase": "0x0000000000000000000000000000000000000000",
-    "difficulty": "0x20",
-    "extraData": "",
-    "gasLimit": "8000000",
-    "nonce": "{{ nonce }}",
-    "mixhash": "0x0000000000000000000000000000000000000000000000000000000000000000",
-    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
-    "timestamp": "0x00"
+  "config": {
+    "chainId": {{ networkid }},
+    "homesteadBlock": 1,
+    "eip150Block": 2,
+    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "eip155Block": 3,
+    "eip158Block": 3,
+    "byzantiumBlock": 4,
+    "constantinopleBlock": 5,
+    "ethash": {}
+  },
+  "nonce": "0x0",
+  "timestamp": "0x5c8f3c67",
+  "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "gasLimit": "0x47b760",
+  "difficulty": "0x10",
+  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "alloc": {
+    "0000000000000000000000000000000000000000": {
```

Comment:
precompiled contracts
https://solidity.readthedocs.io/en/latest/units-and-global-variables.html#mathematical-and-cryptographic-functions